packages/terra-site/src/examples/date-picker/DatePickerDefault.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-default"
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
```

Comment:
Currently the value stored in the hidden input contains only the date part whereas the value to return to consumer in handleChange is the full IOS string. Both places should have a consistent value. Consensus is that we need to discuss with Solution Design to figure out use cases. Let's move forward with this PR and I can log a new issue to address the inconsistent date values.